cost-analyzer/values.yaml

```diff
@@ -758,12 +758,32 @@ awsstore:
   useAwsStore: false
   createServiceAccount: false
 
+federatedETL:
+  # federatedCluster indicates whether this cluster should push data to the Federated store
+  federatedCluster: false 
+  # primaryCluster indicates whether this cluster should load data from the combined section of the Federated store
+  primaryCluster: false 
+  # redirectS3Backup changes the dir of S3 backup to the Federated combined store, for using Thanos-federated data in the Federated ETL
+  # Note S3 backup should be enabled separately for this.
+  redirectS3Backup: false 
+  # useExistingS3Config will attempt to use existing object-store.yaml configs for S3 backup/Thanos as config for the Federated store
+  useExistingS3Config: false
```

Comment:
Should this default to `true`, i.e. is that the recommended value?

Reply:
I don't think so, see reasoning [above](https://github.com/kubecost/cost-analyzer-helm-chart/pull/1728#discussion_r1003549850) in comment with Ajay.